backend/tests/integration/pidstore/minters/test_minters_control_number.py

```diff
@@ -326,3 +325,24 @@ def test_control_number_seminars_without_control_number(inspire_app):
     assert expected_pid_type == result_pid.pid_type
     assert expected_pid_value == result_pid.pid_value
     assert expected_pid_object_uuid == result_pid.object_uuid
+
+
+def test_control_number_minter_proper_pid_management(inspire_app):
+    record = create_record("lit")
+    expected_cn = str(record["control_number"])
+    cn_pid = PersistentIdentifier.query.filter_by(pid_type="lit").one()
+
+    assert cn_pid.pid_value == expected_cn
+
+    data = dict(record)
+    data["control_number"] = f"{expected_cn}1"
```

Comment:
there is `1` at the end, is this correct?

Reply:
Yes, I Intentionally added 1 to change control_number to be sure that it's different to test behavior when CN changes.